dotCMS/src/integration-test/java/com/dotcms/security/apps/AppsAPIImplTest.java

```diff
@@ -806,6 +806,11 @@ public void Test_Save_Secret_Expect_Event_Notification() throws DotDataException
         localSystemEventsAPI.subscribe(AppSecretSavedEvent.class, new AppsSecretEventSubscriber(){
             @Override
             public void notify(AppSecretSavedEvent event) {
+                final AppSecrets appSecrets = event.getAppSecrets();
+                final Map<String, Secret> secrets = appSecrets.getSecrets();
+                secrets.forEach((s, secret) -> {
+                    Assert.assertFalse(isSecretDestroyed(secret.getValue()));
```

Comment:
Codacy found an issue: [Unnecessary use of fully qualified name 'Assert.assertFalse' due to existing static import 'org.junit.Assert.assertFalse'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5884220)